scripts/globals/garrison.lua

```diff
@@ -513,129 +172,246 @@ local spawnNPCs = function(npc)
     end
 end
 
+-- Spawns a mob with the given id for the given zone.
+xi.garrison.spawnMob = function(mobID, zoneData)
+    mob = SpawnMob(mobID)
+    if mob == nil then
+        return nil
+    end
+
+    xi.garrison.addLevelCap(mob, zoneData.levelCap)
+    mob:setRoamFlags(xi.roamFlag.SCRIPTED)
+    table.insert(zoneData.mobs, mobID)
+
+    -- Death listener for tracking win/lose condition
+    mob:addListener("DEATH", "GARRISON_MOB_DEATH", function(mobArg)
+        zoneData.deadMobCount = zoneData.deadMobCount + 1
+        mobArg:removeListener("GARRISON_MOB_DEATH")
```

Comment:
listener should be removed by default I believe.

Reply:
ah okay. Saw this being used elsewhere. Removed.